apps/web/pages/booking/[uid].tsx

```diff
@@ -400,7 +400,7 @@ export default function Success(props: SuccessProps) {
                       </>
                     )}
                     <div className="font-medium">{t("what")}</div>
-                    <div className="col-span-2 mb-6 last:mb-0">{props.bookingInfo.title}</div>
+                    <div className="col-span-2 mb-6 last:mb-0">{eventName}</div>
```

Comment:
I'm not seeing a fallback behaviour, but is this the correct value to display instead, right?

Reply:
yeah